docs/operational_guide/upgrading_m3.md

```diff
@@ -0,0 +1,101 @@
+# Upgrading M3
+
+## Overview
+
+This guide explains how to upgrade M3 from one version to another (e.g. from 0.14.0 to 0.15.0).
+This includes upgrading:
+
+- m3dbnode
+- m3coordinator
+- m3query
+- m3aggregator
+
+## m3dbnode
+
+### Graphs to monitor
+
+While upgrading M3DB nodes, it's important to monitor the status of bootstrapping the individual nodes. This can be monitored using the [M3DB Node Details](https://grafana.com/grafana/dashboards/8126) graph.
+Typically, the `Bootstrapped` graph under `Background Tasks` and the graphs within the `CPU and Memory Utilization` give a good understanding of how well bootstrapping is going.
+
+### Non-Kubernetes
+
+It is very important that for each replica set, only one node gets upgraded at a time. However, multiple nodes can be upgraded across replica sets. 
+
+1) Download new binary (linux example below).
+
+```bash
+wget "https://github.com/m3db/m3/releases/download/$VERSION/m3_$VERSION_linux_amd64.tar.gz" && tar xvzf m3_$VERSION_linux_amd64.tar.gz && rm m3_$VERSION_linux_amd64.tar.gz
```

Comment:
Nit: it's `/download/v$VERSION/m3_$VERSION` (with a `v` in front of first `$VERSION` but not second)